invenio_app_ils/ui/backoffice/src/common/urls.js

```diff
@@ -5,4 +5,6 @@ export const URLS = {
   itemDetails: id => `/items/${id}`,
   loansSearch: '/loans',
   loanDetails: id => `/loans/${id}`,
+  patronsSearch: '/users',
```

Comment:
does this exist?

Reply:
no it does not, but I think it would make sense also to have users' search, WDYT?

Reply:
It depends on the case, if librarians need to navigate in users to perform actions on their behalf maybe would make sense. If it just adds complexity, maybe we can just pop up a searchable list of users when it is needed.